ext/ManoptJuMPExt.jl

```diff
@@ -26,7 +26,7 @@ As the MOI variables are real, this means if the [`representation_size`](@extref
 yields (in product) `n`, this refers to the vectorized point / tangent vector  from (a subset of ``ℝ^n``).
 """
 function MOI.dimension(set::VectorizedManifold)
-    return prod(ManifoldsBase.representation_size(set.manifold))
+    return length(_point_shape(set.manifold))
```

Comment:
Is it (by the way) ok that the actual degrees of freedom are less than this?

For example for `M = Sphere(2)` this dimension is 3 while the manifold-dimension (actual degrees of freedom) are 2.

Reply:
Yes, this is not the dimension in the manifold sense, it's the length of the vectorization

Reply:
Ok, because that sometimes caught us in DiffEq, that they expected DoF and vector dimension to be the same ;)